data/templates/centos-ks

```diff
@@ -53,6 +53,13 @@ sudo
 perl
 libselinux-python
 net-tools
+<% if( undefined !== kvm ) { %>
```

Comment:
Ah, I think I see what you mean Felix. In this case, the option is just a flat value - either defined or not - for what I'm wanting to trigger, as opposed to an intended array of options, possibly empty, so I'm not sure it makes sense to add this as an option to the `_convertOptions` method that's setting those default `[]` values.

Or are you suggesting that we add it as a default "null" value on the install-os.js job in general, so that it's a clear option to be set on the job/task?